### PR TITLE
adding default and loading icons

### PR DESCRIPTION
**Pull request by piyalbasu**
https://user-images.githubusercontent.com/6789586/173400161-e0d4de07-4541-4bd0-a383-3b55470b932d.mov

### CODE REVIEW

File: extension/src/popup/components/manageAssets/SearchAsset/index.tsx [hunk 1]
```diff
@@ -166,10 +166,8 @@ export const SearchAsset = ({ setErrorAsset }: SearchAssetProps) => {
                 ref={ResultsRef}
               >
                 {isSearching ? (
-                  <div className="SearchAsset__overlay">
-                    <div className="SearchAsset__loader">
-                      <Loader />
-                    </div>
```
Comment (piyalbasu) [inline extension/src/popup/components/manageAssets/SearchAsset/index.tsx:172]: removing the dark bg for the loading spinner

File: extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/styles.scss [hunk 2]
```diff
@@ -47,14 +47,21 @@
...
       margin-right: 0.5rem;
     }
 
+    .AccountAssets__asset--error {
+      img {
+        width: 0.4375rem;
+        height: 0.4375rem;
+      }
+    }
+
```
Comment (piyalbasu) [inline extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/styles.scss:64]: I went through and adjusted styles in the Senf/Swap flow that didn't play nice with the new structure. Let me know if anything looks crazy!

File: extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/styles.scss [hunk 3]
```diff
@@ -47,14 +47,21 @@
...
 
-    img {
-      width: 1rem;
-      height: 1rem;
+    .AccountAssets__asset--logo {
+      display: flex;
+      width: 0.875rem;
+      height: 0.875rem;
       border-radius: 1rem;
-      background: var(--pal-brand-primary-on);
```
Comment (piyalbasu) [inline extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/styles.scss:54]: I couldn't quite figure out why this background was needed. Let me know if this needs to come back!

File: extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/styles.scss [hunk 4]
```diff
@@ -47,14 +47,21 @@
     border-radius: 100px;
     padding: 0.25rem 0.5rem;
 
-    img {
-      width: 1rem;
-      height: 1rem;
```
Comment (piyalbasu) [inline extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/styles.scss:52]: the designs had this icon at 14px, so I sized this down in my changes

File: extension/src/popup/components/account/AccountAssets/index.tsx [hunk 5]
```diff
@@ -21,25 +23,40 @@ export const AssetIcon = ({
...
+      className={`AccountAssets__asset--logo ${
+        hasError ? "AccountAssets__asset--error" : ""
+      } ${isLoading ? "AccountAssets__asset--loading" : ""}`}
+    >
+      <img
+        alt={`${code} logo`}
+        src={isXlm ? StellarLogo : imgSrc}
+        onError={() => {
+          if (retryAssetIconFetch) {
+            retryAssetIconFetch({ key: issuerKey, code });
```
Comment (acharb) [inline extension/src/popup/components/account/AccountAssets/index.tsx:44]: ❓ definitely an edge case, but if the retry function fails, then `hasError` will stay as false right?

not sure what the best fix would be 🤔  we could have retryAssetIconFetch return something that could be used to check if failed/succeeded

File: extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/styles.scss (hunk 3 above)
Comment (piyalbasu) [reply extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/styles.scss:54]: Ah right!

File: extension/src/popup/components/account/AccountAssets/index.tsx (hunk 5 above)
Comment (piyalbasu) [reply extension/src/popup/components/account/AccountAssets/index.tsx:44]: Good catch - `setHasError(true)` should always be called. If retry works, it'll re-render and skip right over onError to onLoad

`setIsLoading(false)` actually doesn't really need to be called onError bc if the icon is never found, it just skips down to line 54 where isLoading doesn't have any effect

**Review by acharb — APPROVED** [review]
lgtm!

**Pull request merged**